db-developments/bash/config.sh

```diff
@@ -1,21 +1,12 @@
 #!/bin/bash
-
-# Setting environmental variables
-function set_env {
-  for envfile in $@
-  do
-    if [ -f $envfile ]
-      then
-        export $(cat $envfile | sed 's/#.*//g' | xargs)
-      fi
-  done
-}
+source ../bash_utils/config.sh # expected to be run from db-developments folder
 
 # Setting Environmental Variables
 set_env .env version.env
 DATE=$(date "+%Y-%m-%d")
+set_error_traps
 
-function count {
+function sql_table_summary {
```

Comment:
nice. is the new name in the spirit of expanding this to do more than `count(*)`?